Assets/SuperMultiplayerShooter/Scripts/PubNubUtilities.cs

```diff
@@ -306,18 +317,18 @@ public void UpdatePlayerPosition(PubNub pubnub, int playerID, Vector2 position,
             movementData[5] = velocity.y;
 
             string channelName = playerPositionChannelPrefix + playerID;
-            pubnub.Signal().Message(movementData).Channel(channelName).Async((result, status) =>
+            pubnub.Signal().Message(movementData).Channel(channelName).Execute((result, status) =>
```

Comment:
This makes sense, I was going to say we don't need to `await` for any of the signals but looks like you didn't do that anyway :)